src/pages/404.js

```diff
@@ -1,14 +1,86 @@
 import React from 'react'
+import styled from 'styled-components'
 import SEO from 'components/containers/seo'
 import Layout from 'components/layout/layout'
-import { localize, WithIntl } from 'components/localization'
+import { Header, Text } from 'components/elements/typography'
+import { localize, WithIntl, LocalizedLink } from 'components/localization'
+import Wrapper from 'components/containers/wrapper'
+import Image from 'components/elements/image'
 
-const NotFoundPage = () => (
+const ImageSection = styled(Wrapper)`
+    display: flex;
+    justify-content: center;
+    align-items: center;
+`
+const PageNotFoundContainerInfo = styled.div`
+    width: 50%;
+
+    * {
+        margin: 1.6rem 0;
+    }
+`
+const PageNotFoundContainer = styled.div`
+    display: flex;
+    flex-wrap: nowrap;
+    width: 100%;
+    justify-content: center;
+    align-items: center;
+    margin: 16rem 0;
+`
+const StyledButton = styled(LocalizedLink)`
```

Comment:
as discussed please add a TODO comment here so we remember to move it to button.js later 👍